### PR TITLE
Ways to connect to cluster using kn

### PR DESCRIPTION
**Pull request by vyasgun**
Fixes: [#1251](https://github.com/knative/client/issues/1251)

## Proposed Changes 

- Adding information about `--kubeconfig`, `context`, and `config` options in `kn`

### CODE REVIEW

**Review by dsimansk — CHANGES_REQUESTED** [review]
I'd suggest to completely remove mentions of kn's `--config`flag and file, since there's detailed section about it and it might be confusing in context of connecting to cluster.
https://github.com/knative/docs/blob/mkdocs/docs/client/configure-kn.md

**Comment by dsimansk** [comment]
Thanks @vyasgun! Looks good to me.
/lgtm

File: docs/client/README.md [hunk 1]
```diff
@@ -28,9 +28,14 @@ See [Installing `kn`](install-kn/).
 
 ## Connecting CLI tools to your cluster
 
-After you have installed `kubectl` or `kn`, these tools will search for the `kubeconfig` file of your cluster in the default location of `$HOME/.kube/config`, and will use this file to connect to the cluster.
+After you have installed `kubectl` or `kn`, these tools will search for the `kubeconfig` file of your cluster in the default location of `$HOME/.kube/config`, and will use this file to connect to the cluster. A `kubeconfig` file is usually automatically created when you create a Kubernetes cluster.
+
+You can also set the environment variable `$KUBECONFIG`, and point it to the kubeconfig file.
+
+Using the `kn` CLI, we can specify following options to connect to the cluster.
```
Comment (snneji) [inline docs/client/README.md:35]:
```suggestion
Using the `kn` CLI, you can specify the following options to connect to the cluster:

```

File: docs/client/README.md [hunk 2]
```diff
@@ -28,9 +28,14 @@ See [Installing `kn`](install-kn/).
...
+- `--kubeconfig` option is equivalent to setting the `$KUBECONFIG` environment variable and can be used to point to the kubeconfig file.
```
Comment (snneji) [inline docs/client/README.md:36]:
```suggestion
- `--kubeconfig`: use this option to point to the `kubeconfig` file. This is equivalent to setting the `$KUBECONFIG` environment variable.
```

File: docs/client/README.md [hunk 3]
```diff
@@ -28,9 +28,14 @@ See [Installing `kn`](install-kn/).
...
+- `--context` can be used to specify the name of a context from the existing kubeconfig. (One of the output list of: `kubectl config get-contexts`)
```
Comment (snneji) [inline docs/client/README.md:37]:
```suggestion
- `--context`: use this option to specify the name of a context from the existing `kubeconfig` file. Use one of the contexts from the output of `kubectl config get-contexts`.
```

**Review by snneji — CHANGES_REQUESTED** [review]

**Review by snneji — APPROVED** [review]

**Comment by snneji** [comment]
Yay! Thanks for helping to get this merged!

**Pull request merged**